airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_asserts.py

```diff
@@ -109,30 +109,6 @@ def test_verify_records_schema(configured_catalog: ConfiguredAirbyteCatalog):
         ({"a": "2021-08-10T12:43:15Z"}, {"type": "object", "properties": {"a": {"type": "string", "format": "date-time"}}}, True),
         ({"a": "2018-11-13T20:20:39+00:00"}, {"type": "object", "properties": {"a": {"type": "string", "format": "date-time"}}}, True),
         ({"a": "2018-21-13T20:20:39+00:00"}, {"type": "object", "properties": {"a": {"type": "string", "format": "date-time"}}}, False),
-        # This is valid for postgres sql but not valid for bigquery
```

Comment:
The following format are not supported by Airbyte.

Reply:
Was there any harm in leaving this in?